src/test/java/homework15/ShoppingCartTest.java

```diff
@@ -0,0 +1,68 @@
+package homework15;
+
+import static org.testng.Assert.*;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class ShoppingCartTest extends FuncCart {
+
+    int firstItem = 1;
+    int secondItem = 2;
+
+    @Test(priority = 1)
```

Comment:
```suggestion
    @Test
```
As I see the tests are independent, so need in priority